index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>Portfolio: setup and mobile first</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="#" class="nav-branding">Welcome</a>
+            <div class="humburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+         <section class=" headline">
+           <h2>Hey there. I’m Manar
+           <span class="greenTxt"> Iam a software developer</span></h2>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="socialMedia">
+              <li class="twitter">
+                <img src="/assets/images/twitter.svg" alt="twitter image">
+              </li>
+              <li class="linkedin">
+                <img src="/assets/images/linkedin.svg" alt="linkedin image">
+              </li>
+              <li class="meduim">
+                <img src="/assets/images/medium.svg" alt="medium image">
+              </li>
+              <li class="github">
+                <img src="/assets/images/github.svg" alt="github image">
+              </li>
+              <li class="angleList">
+                <img src="/assets/images/angellist.svg" alt="angleList image">
+              </li>
+            </ul>
+            <div class="moreWorks">
+                <img src="/assets/images/Disabled.png" alt="more work arrow">
```

Comment:
for the view more button :

1. - [ ] please add a link for the image that will navigate you to the next section 
2. - [ ] kindly apply the margins between social links and the button like in Figma

![btn](https://user-images.githubusercontent.com/89282221/184875816-e2b4172f-0e31-4d07-b0d9-0bbdf51f0d26.jpg)